content/blog/2022/06/2022-6-15-svg-icon-migration.adoc

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: "SVG icon migration (breaking changes!)"
+tags:
+- svg-migration
+- community
+- ux
+author: kmartens27
+description: >
+  Starting with Jenkins 2.333, Jenkins core has migrated from `GIF` and `PNG` icons to `SVG` icons.
+---
+
+== Summary
+
+Beginning with Jenkins v2.333 (released on June 15, 2022), Jenkins core has upgraded from `GIF` and `PNG` icons to `SVG` icons.
+Plugins that will be affected by this change are tracked in a link:https://docs.google.com/spreadsheets/d/1PxlgT11_uDyTzPch8zWn3PDxLUIAab21ILmJ17zCzBk/edit#gid=1308179162[spreadsheet] found in link:https://issues.jenkins.io/browse/JENKINS-68251[JENKINS-68251].
+Plugins that do not interact with icons are unaffected by this change.
+
+== Motivation
+
+Updating the Jenkins icons to these previously uploaded SVG icons provides users with a more consistent experience.
+SVG formatting also preserves the image quality during compression, ensuring that icons are displayed properly regardless of scale.
+
+For tracking purposes, link:https://issues.jenkins.io/browse/JENKINS-68251)[JENKINS-68251] is available and contains a link to a link:https://docs.google.com/spreadsheets/d/1PxlgT11_uDyTzPch8zWn3PDxLUIAab21ILmJ17zCzBk/edit#gid=1308179162[spreadsheet] listing plugins and their corresponding states to this update.
+*Make sure that you have used the Plugin Manager to update all plugins before and after updating Jenkins as failing to do so may result in errors.*
+
+
+== Reporting Issues
+
+In the spreadsheet, there is a table labeled “Abandoned plugin, to be discussed”
+
+image:images/post-images/2022-06-svg-migration/abandoned-plugin.png[abandoned plugin]
+
+The table contains plugins that have not had any SCM activity for at least 5 years.
+
+When reviewing this table, if you locate any plugins that are currently used, please reach out to the plugin maintainer to perform this maintenance.
+If the maintainer has indeed abandoned the plugin, link:/doc/developer/plugin-governance/adopt-a-plugin/[adopting the plugin] yourself would be the next best solution.

```

Comment:
Suggesting this since it is an assumption that it is their next best solution. Could be someone else is better for the job, for example. 

```suggestion
If the maintainer has abandoned the plugin, you can link:/doc/developer/plugin-governance/adopt-a-plugin/[adopting the plugin] and manage it yourself.

```

Reply:
Hey @cari-mason, so ultimately, if the plugin is abandoned, _someone_ would need to adopt it but this hypothetical user may be the only person using the plugin and therefore they would _have to_ adopt the plugin and update it to have things continue working properly. All the plugins are maintained outside of CloudBees so it's up to the community to do their part on this.